### ISSUE TITLE
Git Credentials Manager Breaking Change 

### ISSUE THREAD

**Issue by ryang1428** [akiojin/store-git-credential-github-action#104]
As of today, v2.3.0 of the `git-credential-manager` is being installed on our runners, which is resulting with the failure: 

```
Run akiojin/store-git-credential-github-action@v1.7.0
/usr/local/bin/brew tap microsoft/git
/usr/local/bin/brew install --cask git-credential-manager-core
Warning: Not upgrading git-credential-manager-core, the latest version is already installed
/usr/local/bin/git credential-manager-core configure
git: 'credential-manager-core' is not a git command. See 'git --help'.
Error: The process '/usr/local/bin/git' failed with exit code 1
```

According to[ their docs](https://github.com/git-ecosystem/git-credential-manager/blob/main/docs/rename.md#how-to-update) this is expected, and any calls should drop `-core`.

I also confirmed v2.3.0 is now being installed. Our last successful run, it was installing v2.2.2
```
Run brew install --cask git-credential-manager-core
==> Downloading https://github.com/GitCredentialManager/git-credential-manager/releases/download/v2.3.0/gcm-osx-x64-2.3.0.pkg
```

Thanks!

**Comment by akiojin** [comment]
@ryang1428 
Thanks for the information.
Changed the call to `git-credential-manager-core` to `git-credential-manager`.
Fixed in v1.7.1.

**Comment by ryang1428** [comment]
@akiojin thanks!!
